### DISCUSSION TITLE
How to only show context without nav?

### DISCUSSION THREAD

**Discussion by 23WilliamDing** [requarks/wiki#7398] [Help / Questions] [answered]
I wanna ask for help: how to only show context without the left nav?  
This is because sometimes I need to show the left nav, but sometimes I don't need to. Could I realize this by using a parameter in url? Just like, https://docs.requarks.io/install/requirements?showNav=true or https://docs.requarks.io/install/requirements?showNav=false.
@NGPixel any suggestions? THX!
![image](https://github.com/user-attachments/assets/b25fe08f-3c18-4d01-930a-7926f91e3dbb)

**Comment by NGPixel** [comment, marked as answer]
That's not possible no.

**Reply by 23WilliamDing** [reply]
Thank you for your response. I find a solution on my own : 
I inject javascript to identify url, then if it includes 'showNav=false', I use css to hide nav, it works.
![image](https://github.com/user-attachments/assets/bc544637-7571-473f-88e4-036e057aa26c)
